### ISSUE TITLE
Change Email?

### ISSUE THREAD

**Issue by stefankirchner** [openjournals/brief-ideas#104]
A lot of users will change their affiliation with research institutions, change jobs etc. and while used to using official emails like myname @ myuniversity dot edu, there seems to be no way to change the email information one has provided. There is also no way to change it in case one simply has clicked OK too quickly despite a typo in one's email. That is something small that should be fixed. Thanks.

**Comment by arfon** [comment]
Yeah agreed we should do this. We'll likely set up some kind of profile page soon.
